src/portal/src/app/repository/repository-page.component.ts

```diff
@@ -27,13 +26,11 @@ export class RepositoryPageComponent implements OnInit {
   hasProjectAdminRole: boolean;
   hasSignedIn: boolean;
   projectName: string;
-  mode = 'standalone';
```

Comment:
are you test it in admiral env? I remember set default value to `standalong` is to resolve a bug when integrating with admiral.